src/page-modules/departures/__tests__/service-journey-data.fixture.ts

```diff
@@ -21,6 +21,7 @@ export const serviceJourneyFixture: ServiceJourneyData = {
       date: '2023-11-10',
       destinationDisplay: {
         frontText: 'Vestlia via sentrum-Othilienborg',
```

Comment:
Shouldn't this via text be in the `via` array?

Reply:
Fixed